feature-utils/poly-look/rollup.config.mjs

```diff
@@ -16,6 +17,7 @@ export default {
     },
   },
   plugins: [
+    sillyI18n(),
```

Comment:
So we didn't have this before, right? But if we are only using L12n, we don't really need it...

Reply:
I've tried without loading sillyi18n, but rollup complains for `(!) No name was provided for external module '@polypoly-eu/silly-i18n/src/l12n' in output.globals – guessing 'l12n$1'`
I guess we hadn't faced that before as we're using anw the i18n so the sillyi18n was already in rollup config.

Reply:
I would say the only thing you need is name mapping, not to bundle.